recipe/meta.yaml

```diff
@@ -61,23 +61,6 @@ outputs:
         - sktime.transformations
         - sktime.utils
 
-  - name: sktime-all-extras
```

Comment:
This output needs to stay as it provides the `sktime-all-extras` conda package with full set of soft dependencies. Please revert the change.

Reply:
Added it back!